### PR TITLE
Update version to 1.1.0

### PR DESCRIPTION
**Pull request by a4z**
Specify library name and version:  **djinni-support-lib/1.1.0**

This is also a good place to share with all of us **why you are submitting this PR** (specially if it is a new addition to ConanCenter): is it a dependency of other libraries you want to package? Are you the author of the library? Thanks!

---

- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Comment by a4z** [comment]
Close and re-open, since last conan-center-bot message, 1 failure - all successfully, looks like a crash

**Comment by a4z** [comment]
Please help, what is wrong with the PR ?

File: recipes/djinni-support-lib/1.x.x/conanfile.py [hunk 1]
```diff
@@ -102,6 +102,9 @@ def validate(self):
             if self._objc_support or self._jni_support or self._python_support:
                 raise ConanInvalidConfiguration(
                     "C++/CLI is not yet supported with other languages enabled as well. Disable 'with_jni', 'with_objc' and 'with_python' options for a valid configuration.")
+        if self._python_support:
+            if self.settings.os == "Windows":
+                raise ConanInvalidConfiguration("Python on Windowns is not fully yet supported, please see https://github.com/cross-language-cpp/djinni-support-lib/issues.")
```
Comment (ruilvo) [inline recipes/djinni-support-lib/1.x.x/conanfile.py:107]:
```suggestion
                raise ConanInvalidConfiguration("Python on Windows is not fully yet supported, please see https://github.com/cross-language-cpp/djinni-support-lib/issues.")
```

You have a typo. Commit this change and the bot will rebuild and then we'll see.

**Review by ruilvo — CHANGES_REQUESTED** [review]

File: recipes/djinni-support-lib/1.x.x/conandata.yml [hunk 2]
```diff
@@ -2,3 +2,7 @@ sources:
   "1.0.0":
     url: https://github.com/cross-language-cpp/djinni-support-lib/archive/v1.0.0.tar.gz
     sha256: "25f8c648c3bec78a54d5387a792f1d7cd3c073c4477dc50ad2b9c5270a590779"
+sources:
```
Comment (SpaceIm) [inline recipes/djinni-support-lib/1.x.x/conandata.yml:5]:
```suggestion
```

illformed yml

Comment (a4z) [reply recipes/djinni-support-lib/1.x.x/conandata.yml]: thanks!

**Review by jothepro — APPROVED** [review]

**Review by SpaceIm — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Review by ruilvo — APPROVED** [review]

**Review by jgsogo — APPROVED** [review]

**Pull request merged**